web_app/order_books/haiko/docs/block_info.md

```diff
@@ -0,0 +1,29 @@
2612224e412e0a2e430401aac01c736cd035b", "0x2ef83d51bf4a8c7bfae2899e0cc8b31fdb989f715cc1db2be92e5bfc52c209f", "0x221d9ff51aba26ba01176fe3553666384f48bf29d125c20a34cf82704de7cec", "0x3a76adc8d114069ad5c6348e16a288b291fd212fc5d3eed166148690b0bcf80", "0x7217ca04871b7b39c0e390ec99127f0edca6eb8f992b89a4ec1b35bd78035de", "0x25a28b54cee6efe4b6b44f25861d2b1e6e0f18b696dd0998663ed034f4fea9", "0x5708e0fdac5d433fea00163d0d75d0d0f055b392b5055731f76ef4ec1cd8abd", "0x28ffa60afc8c5a045d455b3a5502d6cc83e95e0950f0cffe426a66f81a7c6cb", "0x78292dfeb6cc498e7fba715c6cd7b1a1f5212a5601d4e86a200e87606b791ba", "0x6bd46d7f3bdb4c517aaab3d3967d1a2c6b8e96b0204fb2165dda87ec3360cf", "0x785e61d8819f322a0f2df63a1c09248af35436f43a673dbd00360216ab1c9c9", "0x30b0e27832e38146232aa482bf3a7abe22c5310b141b81fb06b53956e018218", "0x5514a31b2f70b774ffa3bac4b36ab6fc41355fe2dca6c38400c3289adc3bf1a", "0x3a426adb3769ccfa2b55cfe4bacf5add83fca4853089d4b9cc051e1be257cec", "0x74172e5f3c485fa76442353377477e095333b1f5a761e4188fb8d02c6c0d1b2", "0x3a485ff5eae627cbd862c4609dc3fafb028a9f9bb0bc9911fac9d76ff2de9c0", "0x3b66291ef9b42b8ee29b8370071a7a6ea19bf315535df4db730baab370af4fb", "0x24b9c0f27934f22b9c7d8bb1b28b7b640403d8beae241342912865774988a0", "0x351a586cad8bde5ffbde99b7f9ea1d4865439357822608a38b0d8b1479e6358", "0x1a7bcf9e94fe4cd9c85de1ba0e1b9941c42399054d39c4f519f9fba8dc223de", "0x21159bcfb54ef6bb52ad93fa631aa36c2424a9747e434818c4bb4722582a713", "0x188b884c9e507a007c639c4547984b90c1cecc0cf2221e156b1b075b3fb7655", "0x56d15b44d0640869cd9be489497264e00dd73df5455b003078ff56a98aba149", "0x2eddfaa325c18d8845ff721b75808216d6ef391160d67c609bc30f59393bf9", "0x74cb241a323c476dc2287c1754aa75b6156f21e9f93f6df364c97b977683462", "0x13fed22d5c67af47f9e84e072bf782461f6f682d14235803fd2dc178c1e383e", "0x439e88605bb0a6e172c3d599257f38140018ea81910a6480ff1b63638ebd268", "0x1e795281ec4cc4d3e76902a8e713b8355a52fb9c46883169e3dc0e509e374f6", "0x1ac511e20f9ab384e7c1cb7706c284d06ee1a05d8131ad7cc20341e9c344e32", "0x2c96bd9bdcd0173d8dd08cb735bb234ba082acab6b956255b627d7e944f9817", "0xddfd7272311b588c92adf91d75d7352177a55728104b7c745b716e3c2f54b2", "0x4998af8695aea3e9301f1db32f89101d6725a2b04442b600bf2b71fd9b80449", "0x4d8f684a8c58df7ef4d1fdb7643d5b34d488363e246d9bf575b7323f0c372f6", "0x78625b74ad44bb049b0344eb16be7abef9de9aa5fd212ddfee196e8505bba0", "0x1cfe4e308ce443529962e17d877994ca647c42cc53ff303e02085c70f19f3a5", "0x7657f6571b83fcdd524d672e4ba178432a60eadbf9683399557a54c3526b1d3", "0x34da350253263c647dbd153c9eb6186c06f2bfd7b32b8c70786774b14a9157a", "0x23a0ec2f7faecb95595d19ed28dd9c320835d1aaee7b1d52c1b93d56b92e7cd", "0x44b8afa4dbfd64240f380390653a59e12f974aee68b52a20cacd3681d143304", "0x5eb267c6da61f84812fa2776ddcc7f19b920fd9e50cd3096c88cb053acb27f9", "0x6011b791c3c4e81f3e897f216e524680ebd11686201d9554e2cee2aee0e9ecc", "0x2a0e8e734bbbf135c7eb094b4ac9566e4b925c8cc155171744ee6c7e175a00f", "0x1c429d2c8fcf7687063cfdb951afa3da8cb0ffae8fd560058c1813a9e1fbcb8", "0x121aaba2894a63cce81acabb559eb2c114f4859dc5803e0b91349ec398d2eb0"]
```

Comment:
Feel free to cut down number `number_of_trasaction` to 5. to make it more readable